ros/api/v1/hosts.py

```diff
@@ -197,9 +197,10 @@ def build_system_filters():
             modified_states = []
             for state in states:
                 state = state.capitalize()
-                modified_states.append(state)
```

Comment:
Previous and current changes both would work the same way. Is there any reason why we need this change?

Reply:
It is more of an optimization, looks like we can abort before an append operation

Reply:
But append operation has constant time complexity.

Reply:
I think we are doing unnecessary append to the list even if its invalid?